src/state/network/actions.js

```diff
@@ -87,7 +87,8 @@ export function detectNormandyAdmin() {
       // if this succeeds, then the normandy admin is available
       await Promise.race([adminPromise, timeoutPromise]);
       dispatch({ type: REQUEST_SUCCESS, requestId });
-      dispatch({ type: NETWORK_NORMANDY_ADMIN_AVAILABLE, available: true });
+      // dispatch({ type: NETWORK_NORMANDY_ADMIN_AVAILABLE, available: true });
+      dispatch({ type: NETWORK_NORMANDY_ADMIN_AVAILABLE, available: false });
```

Comment:
Is this leftover debugging, or was this change intentional?

Reply:
The commented out code is just sloppy-branch left-overs. I'll deal with all of that once I know this branch is worth completing.

Reply:
No wait! Sorry. I take that back. I was referring to a different [pr](https://github.com/mozilla/delivery-console/pull/836).

I'll clean this up.